staging/src/k8s.io/kubectl/pkg/describe/describe_test.go

```diff
@@ -4668,22 +4699,86 @@ URL:	http://localhost
 				"uid":               "00000000-0000-0000-0000-000000000001",
 				"dummy3":            "present",
 			},
+			"itemsStrings": []interface{}{
+				"dummy4", "dummy5",
+			},
+			"itemsWithOneObject": []interface{}{
+				map[string]interface{}{
+					"itemBool": true,
+					"itemInt":  42,
+				},
+			},
+			"itemsWithTwoObjects": []interface{}{
+				map[string]interface{}{
+					"itemBool": true,
+				},
+				map[string]interface{}{
+					"itemInt": 42,
+				},
+			},
 			"items": []interface{}{
+				"dummy4",
 				map[string]interface{}{
 					"itemBool": true,
 					"itemInt":  42,
 				},
+				map[string]interface{}{
+					"itemBool": false,
+					"itemInt":  -1,
+				},
+				map[string]interface{}{
+					"itemBool": true,
+				},
+				map[string]interface{}{
+					"itemInt": 1,
+				},
+				map[string]interface{}{
+					"itemInt":         2,
+					"itemStringArray": []interface{}{"dummy5", "dummy6"},
+				},
+				map[string]interface{}{
+					"itemObject": map[string]interface{}{
+						"itemInt": 4,
+					},
+					"itemObjectTwo": map[string]interface{}{
+						"itemInt":  4,
+						"itemBool": true,
+					},
+				},
+				map[string]interface{}{
+					"itemObjectArray": []interface{}{
+						map[string]interface{}{
+							"itemInt": 4,
+						},
+						map[string]interface{}{
+							"itemBool": true,
+						},
+					},
+				},
+				map[string]interface{}{
+					"itemObjectArrayWithOneElement": []interface{}{
+						map[string]interface{}{
+							"itemInt": 4,
+						},
+					},
+				},
+				[]interface{}{
+					5, 6, 7,
+				},
+				"dummy7",
+				5,
 			},
 			"url":    "http://localhost",
 			"status": "ok",
 		},
 	}
-	printUnstructuredContent(w, LEVEL_0, obj.UnstructuredContent(), "", ".dummy1", ".metadata.dummy3")
+	printUnstructuredContent(w, LEVEL_0, obj.UnstructuredContent(), false, "", ".dummy1", ".metadata.dummy3")
 	output := out.String()
 
 	for _, test := range testCases {
 		if len(test.expected) > 0 {
 			if !strings.Contains(output, test.expected) {
+				t.Logf("%s", output)
```

Comment:
You don't need this, it's already printed below in %q

Reply:
This prints multipline output while Errorf below escapes new lines due to %q

Reply:
Yes but do we need this?
You verified the output and then we can drop the output.